tools/custom/raw-bytecode-analyser/README.md

```diff
@@ -0,0 +1,75 @@
+# Hedera Smart Contract Bytecode Analyzer
+
+This Python script analyzes the bytecode of Smart Contracts on the Hedera network.
+
+## Requirements
+
+- Python 3.6 or higher
+- `requests` library
+- `evmdasm` library
+
+To install the required Python libraries, run:
+
+```bash
+pip install requests evmdasm
+```
+
+## Usage
+
+The script can be run from the command line with the following arguments:
+
+- `contract_id`: The ID of the smart contract whose bytecode you want to analyze.
+- `--previewnet`: Optional flag to use the Previewnet Mirror Node URL.
+- `--testnet`: Optional flag to use the Testnet Mirror Node URL.
+
+If no network flag is specified, the script defaults to using the Mainnet Mirror Node URL.
+
+### Basic Command
+
+```bash
+python detect_token_creation_with_secp_key.py <contract_id>
+```
+
+### Examples
+
+1. **Analyze a Contract on Mainnet**
+
+    ```bash
+    python detect_token_creation_with_secp_key.py 0.0.123456
+    ```
+
+2. **Analyze a Contract on Testnet**
+
+    ```bash
+    python detect_token_creation_with_secp_key.py 0.0.123456 --testnet
+    ```
+
+3. **Analyze a Contract on Previewnet**
+
+    ```bash
+    python detect_token_creation_with_secp_key.py 0.0.123456 --previewnet
+    ```
+
+## Features
+
+The script performs the following checks on the smart contract bytecode:
```

Comment:
I would include an overview of how the detection works, and maybe a brief introduction to `evmdasm`.

Reply:
I've added "How detection works" section.